docs/pages/desktop-access/getting-started.mdx

```diff
@@ -148,12 +148,15 @@ certificate-based smart card authentication, and ensuring RDP is enabled.
 
 ### Export the Teleport CA
 
-<Admonition type="note" title="Requires Existing Cluster">
-  The following step requires an existing cluster. If you don't already have a
-  Teleport cluster up and running, see our general [Getting
-  Started](../getting-started.mdx) guide.
+The following step requires an existing cluster. If you don't already have a
+Teleport cluster up and running, see our general [Getting
+Started](../getting-started.mdx) guide.
+
+<Admonition type="note" title="User CA Rotation">
+These steps will need to be repeated if Teleport's user certificate authority is rotated.
```

Comment:
```suggestion
These steps will need to be repeated if the Teleport's user certificate authority is rotated.
```

Reply:
hmm. I don't think your suggestion is grammatically correct here.

Reply:
Sorry, I'd meant to change that to "if the Teleport user's," but noticed that the original is fine as it is. Please ignore!